src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -121,15 +121,15 @@
 				<ul class="scroller h-100">
 					<!-- Conversations List -->
 					<template v-if="!isSearching">
+						<Hint v-if="initialisedConversations && filteredConversationsList.length === 0" :hint="t('spreed', 'No matches found')" />
```

Comment:
Maybe we can include the filter that is currently applied?
And maybe also a button to remove the filter?

Reply:
We can add applied filter as NcAppNavigationCaption in that case, I guess
But title-name prop change should be considered, as we haven't merged lib yet

Reply:
Should it be kept bold and blue ?
![image](https://github.com/nextcloud/spreed/assets/84044328/15ec1e8c-33b5-407f-bf9a-9c0388b09c14)

Reply:
I thought about

```
### Unread mentions
No matches found
```